workflows/prognostic_c48_run/runtime/loop.py

```diff
@@ -217,6 +255,7 @@ def __init__(
         self._log_debug(f"States to output: {self._states_to_output}")
         self._prephysics_stepper = self._get_prephysics_stepper(config, hydrostatic)
         self._postphysics_stepper = self._get_postphysics_stepper(config, hydrostatic)
+        self._radiation_wrapper = self._get_radiation_wrapper(config, namelist)
```

Comment:
Why is it a "wrapper" rather than a "stepper"? Can you make it a stepper?

Reply:
Yeah, it can be a `Stepper` -- will do